18F/identity-idp
# Fix spinner button animation for long action text

## 🛠 Summary of changes

Fixes an issue where SpinnerButton may appear incorrect when using longer `action_message` text. The issue is that the spinner button's container width will grow to accommodate the longer text, but this growth causes the spinner animation to be positioned incorrectly, since it is horizontally centered.

## 📜 Testing Plan

1. Go to http://localhost:3000/components/inspect/spinner_button/workbench?action_message=This+is+a+long+action+message.
2. Click button
3. Observe spinner animation is centered in the button, and the action message appears after 15 seconds

## 👀 Screenshots

Before|After
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/653f13f7-ddd7-4e9b-a819-8e58cf153658)|![image](https://github.com/18F/identity-idp/assets/1779930/94d32f42-f9ab-41cf-b18c-44bd50369b26)

Note the issue is horizontal centering:

![image](https://github.com/18F/identity-idp/assets/1779930/e84a091e-4472-447f-823a-e8a136465033)
